wrap/docs/doc_template.py

```diff
@@ -0,0 +1,72 @@
+import xml.etree.ElementTree as ET
```

Comment:
Is this being used now? I know Matthew worked in this for quite a while

Reply:
Nope, but I expect to resurrect this soon :)